msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/DiscussionForumTool.java

```diff
@@ -3261,6 +3262,10 @@ private DiscussionTopicBean getDecoratedTopic(DiscussionTopic topic)
 			  }
 		  }
 
+		// Determine if we should display authors' userIDs or their anonIDs
+		boolean useAnonymousId = isUseAnonymousId(topic);
+		Map<String, String> userIdAnonIdMap = null;
```

Comment:
While this will get set below, it is safer to assign Collections.emptyMap() here instead of null.